rllib/algorithms/impala/impala_torch_policy.py

```diff
@@ -222,6 +222,8 @@ def __init__(self, observation_space, action_space, config):
             max_seq_len=config["model"]["max_seq_len"],
         )
 
+        ValueNetworkMixin.__init__(self, config)
```

Comment:
why did we add the value network mixin here? what were we doing before?

Reply:
For IMPALA, nothing. We did not need to compute any VF_PREDs, ever. Because we were doing this in the loss function, using a `model.value()` call.